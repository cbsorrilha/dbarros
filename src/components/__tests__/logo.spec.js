import React from "react"
import renderer from "react-test-renderer"
import Logo from '../logo';


describe('Snapshot testing the Logo', () => {
    it("renders correctly", () => {
        const props = {}
        const tree = renderer.create(<Logo {...props} />).toJSON()
        expect(tree).toMatchSnapshot()
    })
})
