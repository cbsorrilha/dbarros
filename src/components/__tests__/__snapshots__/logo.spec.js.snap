// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot testing the Logo renders correctly 1`] = `
<h1
  className="sc-AykKD kFUErR"
>
  <a
    className="sc-AykKC ewilfx"
    href="/"
  >
    <img
      alt="dbarros logo"
      src="test-file-stub"
    />
  </a>
</h1>
`;
